ipod-shuffle-4g.py

```diff
@@ -469,7 +469,7 @@ def construct(self, tracks):
             playlist.populate(i)
             construction = playlist.construct(tracks)
             if playlist["number_of_songs"] > 0:
-                if playlist["listtype"] == 3:
+                if playlist["listtype"] == PlaylistType.PODCAST.value:
```

Comment:
Why do you use .value here?

Reply:
I would use IntEnum here and maybe rethink about the filetype enum. Is that filetype enum even required? It looked to me that it makes things just more complicated, but I might be wrong

Reply:
Sounds good. Actually, in researching how to make enums comparable to ints I came across reasons not to do so (enums aren't ints), but I can instead convert the `listtype` value to an enum.
`if PlaylistType(playlist["listtype"]) == PlaylistType.PODCAST`
I might like that better actually.

I like the FileType enum because it more tightly groups the file extensions with the FileType they go with and it adds consistency that the database values that are a limited range of values should be handled as those values.

Reply:
Using `IntEnum` does not work?

Reply:
It _can_ work, but while researching how to do so I came across reasons not to do so and I think it's clearer to directly compare the enum rather than mixing enums and ints.